### PR TITLE
Shorten logging class name for CustomListenerHost

### PR DESCRIPTION
**Pull request by mikeparker**
Also fix a few formatting issues.

Signed-off-by: Mike Parker <michael.parker@docker.com>

### CODE REVIEW

File: src/HttpOverStream.Server.Owin/CustomListenerHost.cs [hunk 1]
```diff
@@ -22,13 +21,13 @@ public class CustomListenerHost : IDisposable
...
         private bool _disposed;
-        static byte[] _eol = Encoding.ASCII.GetBytes("\n");
+        static readonly byte[] _eol = Encoding.ASCII.GetBytes("\n");
 
         private CustomListenerHost(IListen listener, Func<IDictionary<string, object>, Task> app, IAppBuilder builder)
         {
             _listener = listener;
             _app = app;
-            _logger = builder.CreateLogger<CustomListenerHost>();
+            _logger = builder.CreateLogger("HttpOS.Owin.CLH");
```
Comment (mat007) [inline src/HttpOverStream.Server.Owin/CustomListenerHost.cs:30]: Would `HttpOverStream` work? 🤔

Comment (mikeparker) [reply src/HttpOverStream.Server.Owin/CustomListenerHost.cs:30]: What just HttpOverStream? It doesnt tell you which class the message came from, but maybe thats fine?

Comment (mat007) [reply src/HttpOverStream.Server.Owin/CustomListenerHost.cs:30]: Or not, that's probably fine as it is. It's just `HttpOS.Owin.CLH` is a little difficult to say out loud 😬

**Review by mat007 — APPROVED** [review]

File: src/HttpOverStream.Server.Owin/CustomListenerHost.cs (hunk 1 above)
Comment (mikeparker) [reply src/HttpOverStream.Server.Owin/CustomListenerHost.cs:30]: I agree, its ugly. However, the most important thing is that we know without doubt where a log line originated from. These are for making diagnostics as specific as possible, not for readability. It wastes a huge amount of time if you have a debug issue but you're not sure where the error actually came from and you need to tweak the logs and rebuild. Hopefully no-one will need to say it out loud...

**Pull request merged**